src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -183,6 +188,9 @@ private void initializeRamseteCommands() {
               ramsetes[i],
               new CombinedIntakeCDSForwardCommand(intakeSubsystem, cdsSubsystem, shooterSubsystem));
     }
+    parallels[0] =
+        parallels[0].deadlineWith(new InstantCommand(climbSubsystem::deployHA, climbSubsystem));
+
```

Comment:
After thinking about this, this will break the ramsete command because the deployHA will finish first and cause the others in the deadline to stop.